_includes/latest-content-header.ja.html

```diff
@@ -1,3 +1,3 @@
 <div class="note">
-  <p>これはPGroonga 2.X and 3.X用のドキュメントです。古いPGroongaを使っているなら<a href="../{{ page.upper_level }}v1/{{ page.url | remove_first: "/ja/" }}">PGroonga 1.xのドキュメント</a>を見てください。</p>
+  <p>これはPGroonga 2.X、3.X、4.x用のドキュメントです。古いPGroongaを使っているなら<a href="../{{ page.upper_level }}v1/{{ page.url | remove_first: "/ja/" }}">PGroonga 1.xのドキュメント</a>を見てください。</p>
```

Comment:
```suggestion
  <p>これはPGroonga 2.X、3.X、4.X用のドキュメントです。古いPGroongaを使っているなら<a href="../{{ page.upper_level }}v1/{{ page.url | remove_first: "/ja/" }}">PGroonga 1.xのドキュメント</a>を見てください。</p>
```

「PGroonga 2.0.0以降」にしちゃってもいいかも。